blocks/api/categories.js

```diff
@@ -27,3 +30,104 @@ const categories = [
 export function getCategories() {
 	return categories;
 }
+
+/**
+ * Register a new block category
+ *
+ * @param {Array} cat e.g {slug: 'custom', title: __('Custom Blocks')}
+ *
+ * @return {Array} Block categories
+ *
+ */
+export function registerCategory( cat ) {
```

Comment:
Minor: Abbreviations are overrated in my opinion. Don't let a few characters dissuade you from clarity, especially since it's all minified in the end anyways.

Cats are for meowing 😸

Reply:
Hi, sorry for the delay. I was on holiday. Right, Let's use the full-lenght definition. Moreover, I'm also allergic so better don’t bother cats for this purpose. :)